### PR TITLE
[dockers] Rename 'docker-snmp-sv2' to 'docker-snmp'

### PR DESCRIPTION
**Pull request by jleveque**
The `-sv2` suffix was used to differentiate SNMP Dockers when we transitioned from "SONiCv1" to "SONiCv2", about four years ago. The old Docker materials were removed long ago; there is no need to keep this suffix. Removing it aligns the name with all the other Dockers.

Also edit Monit configuration to detect proper snmp-subagent command line in Buster, and make snmpd command line matching more robust.

### CODE REVIEW

**Comment by qiluo-msft** [comment]
> check process snmp_subagent matching "python3.6 -m sonic_ax_impl"

This will not work on buster image.

---
Refers to: dockers/docker-snmp/base_image_files/monit_snmp:10 in 88c6bfd. [](commit_id = 88c6bfd68ea7a3f3c7a5ade4ec14684ffa76696f, deletion_comment = False)

File: platform/generic/rules.mk [hunk 1]
```diff
@@ -3,7 +3,7 @@ include $(PLATFORM_PATH)/onie-image.mk
 
 SONIC_ALL += $(DOCKER_DATABASE) \
          $(DOCKER_LLDP_SV2) \
```
Comment (qiluo-msft) [inline platform/generic/rules.mk:5]: >DOCKER_LLDP_SV2 [](start = 11, length = 15)

Do you want to fix docker-lldp-sv2 in the same PR?

Comment (jleveque) [reply platform/generic/rules.mk:5]: I was going to do each container in separate PRs.

**Comment by jleveque** [comment]
> > check process snmp_subagent matching "python3.6 -m sonic_ax_impl"
> 
> This will not work on buster image.

@qiluo-msft: Thanks for the heads-up. It's not really related to this PR, but I can make the change here, if you'd like.

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest mellanox please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

**Pull request merged**